src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -252,11 +252,11 @@ public String quit() {
   }
 
   /**
-   * Test if the specified keys exist. The command returns the number of keys existed Time
-   * complexity: O(N)
+   * Test if the specified keys exist. The command returns the number of keys exist.
+   * Time complexity: O(N)
    * @param keys
-   * @return Integer reply, specifically: an integer greater than 0 if one or more keys existed 0 if
-   *         none of the specified keys existed
+   * @return Integer reply, specifically: an integer greater than 0 if one or more keys exist
```

Comment:
Add a full stop at the end.

Reply:
I suppose the full stop should be at the end of the next line (258), but the comma - at the end of this line (258), right?

Reply:
What we need is a proper punctuation after 258's `exist`. So I suggested a full stop. But your suggestion also works.